packages/tocco-theme/src/ToccoTheme/variable/mutable-imports.scss

```diff
@@ -0,0 +1,15 @@
+@import '../../../../entity-browser/src/components/DetailView/DetailView.scss';
```

Comment:
I'd also prefer if we could get rid of this file. This breaks IMHO module concepts. Would be nice if this file could be generated at runtime.